src/pip/_internal/locations/__init__.py

```diff
@@ -134,7 +133,7 @@ def _looks_like_red_hat_scheme() -> bool:
     from distutils.command.install import install
     from distutils.dist import Distribution
 
-    cmd: Any = install(Distribution())
+    cmd: install = install(Distribution())
```

Comment:
I’m just curious why this annotation is even needed. Shouldn’t the return type be inferred…?

Reply:
Yes, you are right. I checked with both our existing `mypy` calls and with the full `mypy` in #13599, and in both cases `reveal_type` correctly gives us the install command class as the type.

I've added a commit.